frontend/views/components/Chatroom/ChatMain.vue

```diff
@@ -235,7 +235,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import "../../../assets/style/_variables.scss";
+@import "_variables.scss";
```

Comment:
This is way simpler, but it's not very intuitive. I believe we should follow the same approach as in JS, using alias. Something like:

```scss
@import '@css-variables'
// or a more correct name:
@import '@css-theme'
```

Reply:
Maybe, I'll check and see if `@assets` works